pkg/controller/importconfig/importconfig_controller.go

```diff
@@ -91,7 +91,7 @@ func (r *ReconcileImportConfig) Reconcile(ctx context.Context, request reconcile
 	switch mode {
 	case constants.KlusterletDeployModeDefault:
 		yamlcontent, err = bootstrap.NewKlusterletManifestsConfig(
-			operatorv1.InstallModeDefault,
```

Comment:
Just to confirm, the install mode origin is from this line of code: https://github.com/stolostron/managedcluster-import-controller/blob/94cefd12dc428f999991dfeabc75fdac6b75ba2f/pkg/helpers/helpers.go#L642

`mode, ok := cluster.Annotations[constants.KlusterletDeployModeAnnotation]` From this annotation, we still only have 2 modes right? 

Only now we render with `Mode:Singleton` when we extract `Default` from the annotation?

Reply:
yes

Reply:
let me think about it, otherwise we need ui to update

Reply:
updated, if annotation is not set, use the singleton by default, otherwise read the annotation accordingly

Reply:
Got it.